test/fixture/node-paths/modules/nested/foo.js

```diff
@@ -0,0 +1,3 @@
+module.exports = function() {
+	return 'bar';
+}
```

Comment:
Needs a semi-colon after the `}`. And could you insert a linebreak at the end of the file?